### PR TITLE
Add renders for kubernetes

### PR DESCRIPTION
**Pull request by oded-dd**
tests are still missing and README should be added with the SHA, but other then that I think it is ready.

Please add your notes

### CODE REVIEW

File: utils/package.dhall [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{ LogLevel = ./LogLevel.dhall
```
Comment (ari-becker) [inline utils/package.dhall]: Recommend renaming `utils` to `primitives`

File: utils/package.dhall [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+, Image = ./Image.dhall
+, Duration = ./Duration.dhall
+, Resources =
+    { Compute = ./Compute.dhall
+    , Memory = ./Memory.dhall
+    , Storage = ./Storage.dhall
+    }
```
Comment (ari-becker) [inline utils/package.dhall]: if resources are nested then they should go into `primitives/resources/package.dhall`

File: utils/LogLevel.dhall [hunk 3]
```diff
@@ -0,0 +1,16 @@
+let LogLevel = < All | Debug | Info | Warn | Error | None > : Type
```
Comment (ari-becker) [inline utils/LogLevel.dhall]: I think this was specific to some of our work on, I don't recall off the top of my head, either prometheus or fluentd. It's difficult to say exactly which log levels are going to be valid in general for anything - maybe a project doesn't support `All`, for instance, but by using `LogLevel` here then downstream could erroneously select for `All`.

I think `utils/LogLevel.dhall` should be removed.

Comment (oded-dd) [reply utils/LogLevel.dhall]: Disagree this is not a straightforward render it is useful for a lot of cases.

**Pull request merged**